app/components/widgets/AlertBox.vue

```diff
@@ -31,7 +31,10 @@
         <span class="left-accordion__title">{{ alertName(alert) }}</span>
       </div>
       <div class="left-accordion__input" v-if="wData.settings[alert]">
-        <validated-form  @input="save()"><toggle-input v-model="wData.settings[alert].enabled" /></validated-form>
+        <validated-form v-if="alert !== 'facemasks'" @input="save()"><toggle-input v-model="wData.settings[alert].enabled" /></validated-form>
+        <validated-form v-if="alert === 'facemasks'" @input="handleFacemaskInput()">
```

Comment:
See comment about extracting above. This also impacts the previous line.

Reply:
extracting doesnt work here since it's in a for loop